jenkins-plugin/src/main/resources/org/jenkinsci/plugins/pipeline/maven/GlobalPipelineMavenConfig/config.jelly

```diff
@@ -27,8 +27,14 @@ THE SOFTWARE.
          xmlns:f="/lib/form" xmlns:c="/lib/credentials">
 
     <f:section title="${%Pipeline Maven Configuration}">
-        <f:entry title="${%DAO}">
-            <pre><code>${instance.dao.toPrettyString()}</code></pre>
```

Comment:
removing this line is the reason of the drop

Reply:
the issue I have with this and how it was done. 
We were forcing the `getDao()` and eventually possible error while users didn;t have yet configured yet jdbcUrl.

Reply:
we can restore it but it will call `getDao()` even before users have configured iit.

Reply:
or calling `instance.getDaoSomething()`  which will check if dao is null or not and return "" or prettyString ?

Reply:
done via 0f736cc7edca57aebf8a7af21b8f19ee30627ca7